common/hugo/hugo.go

```diff
@@ -61,6 +61,12 @@ func (i Info) Version() VersionString {
 	return CurrentVersion.Version()
 }
 
+// GoVersion returns the version of Go that the hugo binary is running with.
```

Comment:
* I think it would be better (or at least more consistent) if you just add GoVersion as a field and add it to the constructor.
* I think it's more precise to say "that the hugo binary is built with"...
* Also, the commit message is not totally in line with the commit message guidelines (does not start with a capital letter after the prefix).

Reply:
feedback accommodated, thanks for the speedy review!